src/Illuminate/Contracts/Events/Dispatcher.php

```diff
@@ -7,7 +7,7 @@ interface Dispatcher
     /**
      * Register an event listener with the dispatcher.
      *
-     * @param  \Closure|string|array  $events
+     * @param  \Closure|\Illuminate\Events\QueuedClosure|string|array  $events
```

Comment:
This cannot go in the contract, because the contracts repo cannot know about the events concetions.

Reply:
We probably need a contract for this class, to allow us to reference it here, and in all phpdoc in this PR.